components/event-publisher-proxy/pkg/legacy-events/legacy.go

```diff
@@ -72,20 +76,17 @@ func (t Transformer) checkParameters(parameters *apiv1.PublishEventParametersV1)
 	return &apiv1.PublishEventResponses{}
 }
 
-func (t Transformer) TransformsLegacyRequestsToCE(writer http.ResponseWriter, request *http.Request) *cev2event.Event {
-
+func (t Transformer) TransformLegacyRequestsToCE(writer http.ResponseWriter, request *http.Request) *cev2event.Event {
 	// parse request body to PublishRequestV1
 	if request.Body == nil || request.ContentLength == 0 {
 		resp := ErrorResponseBadRequest(ErrorMessageBadPayload)
 		writeJSONResponse(writer, resp)
 		return nil
 	}
-	appName := ParseApplicationNameFromPath(request.URL.Path)
-	var err error
+
 	parameters := &apiv1.PublishEventParametersV1{}
 	decoder := json.NewDecoder(request.Body)
-	err = decoder.Decode(&parameters.PublishrequestV1)
-	if err != nil {
+	if err := decoder.Decode(&parameters.PublishrequestV1); err != nil {
```

Comment:
Is it possible to change the variable `PublishrequestV1` to `PublishRequestV1` in this PR?

Reply:
I agree this should change, but please let's not extend the scope of this ticket.